src/chocolatey.resources/chocolatey.resources.csproj

```diff
@@ -115,6 +115,9 @@
     <EmbeddedResource Include="helpers\functions\Set-EnvironmentVariable.ps1" />
     <EmbeddedResource Include="helpers\functions\Test-ProcessAdminRights.ps1" />
   </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="helpers\functions\Generate-BinFile.ps1" />
+  </ItemGroup>

```

Comment:
Actually, looks like there are a few outside the main item group.  What do you want me to do here?